### PR TITLE
Adds dxdaomesa module's contracts.json to package_data

### PR DESCRIPTION
**Pull request by coinyon**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -27,7 +27,12 @@
     packages=find_packages('.'),
     package_data={
         # TODO: Investigate if it's needed. rotkehlchen.spec is where files seem to be copied
-        'rotkehlchen': ['data/*.json', 'data/*.meta', 'data/*.db'],
+        'rotkehlchen': [
+            'data/*.json',
+            'data/*.meta',
+            'data/*.db',
+            'chain/ethereum/modules/dxdaomesa/data/contracts.json',
```
Comment (LefterisJP) [inline setup.py:34]: Can you keep it in sync with rotkehlchen.spec and edit the comment to make sure these 2 are kept in sync?

Comment (coinyon) [reply setup.py:41]: Remove wildcards and added comments

**Review by LefterisJP — COMMENTED** [review]
Hey thanks @coinyon. Could you combine commit 1 and commit 4 as they are basically touching exactly same code?

**Comment by coinyon** [comment]
Sure, done

**Review by LefterisJP — APPROVED** [review]
Thanks @coinyon. LGTM.

**Pull request merged**